templates/medium/eap-wildfly/camel-test-cdi/README.md

```diff
@@ -0,0 +1,91 @@
+WildFly Camel CDI Web Application
+=============
+
+This is a template Apache Camel CDI application for the WildFly Camel subsystem. 
+
+This project is setup to allow you to create a Apache Camel CDI application, which can be deployed to an application
+server running the WildFly Camel subsystem. An example CDI Camel Route has been created for you, together with an Arquillian
+integration test.
+
+Prerequisites
+=============
+
+* Minimum of Java 1.7
+* Maven 3.2 or greater
+* JBoss EAP 6.4
+
+
+Getting started
+---------------
+
+1. Install JBoss Fuse on your application server
+
+2. Configure a $JBOSS_HOME environment variable to point at your application server installation directory
```

Comment:
ok, can we at least modify the "Getting started" into "Getting started for command-line"?